Dockerfile

```diff
@@ -28,8 +28,10 @@ FROM builder as production
 #RUN poetry config virtualenvs.create false \
 #    && poetry install --no-interaction
 # set my port on container to 5001
-EXPOSE 5001
-CMD ["gunicorn", "--bind", "0.0.0.0:5001", "--log-level=debug", "app:create_app()"]
+# Removed expose and port bind for 5001 due to port binding issue on Heroku
+# EXPOSE 5001
+# CMD ["gunicorn", "--bind", "0.0.0.0:5001", "--log-level=debug", "app:create_app()"]
+CMD ["gunicorn", "--log-level=debug", "app:create_app()"]
```

Comment:
I expected `CMD gunicorn --log-level=debug --bind 0.0.0.0:$PORT app:create_app()` 

See https://devcenter.heroku.com/articles/container-registry-and-runtime#get-the-port-from-the-environment-variable

Did it really deploy successfully? Heroku dashboard/history didn't show any problems? I guess it's possible that Heroku just happened to assign it gunicorn's default port of 8000